### PR TITLE
since we allow calling save() without pass any argument, we need this…

### PR DESCRIPTION
**Pull request by 66eli77**
I should have tested the PR...

### CODE REVIEW

File: kolibri/core/assets/src/api-resource.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ class Model {
 
   set(attributes) {
     // force IDs to always be strings - this should be changed on the server-side too
-    if (this.resource.idKey in attributes) {
+    if (attributes && this.resource.idKey in attributes) {
```
Comment (rtibbles) [inline kolibri/core/assets/src/api-resource.js:218]: Ah, this might explain the issue that @ralphiee22 was having also.

@ralphiee22 probably good to revert the change we made, and pull in this from master when it is merged.

**Comment by rtibbles** [comment]
@jamalex asked you if you tested, and you said YES!

**Comment by 66eli77** [comment]
I should test this pr before merged indeed, but what @jamalex asked is a different piece of code
